README.md

```diff
@@ -12,7 +12,8 @@ The design of TiKV ('Ti' stands for titanium) is inspired by some great distribu
 
 If you're interested in contributing to TiKV, or want to build it from source, see [CONTRIBUTING.md](./CONTRIBUTING.md).
 
-![cncf_logo](images/cncf.png)
+![cncf_logo](images/cncf.png#gh-light-mode-only)
```

Comment:
How about using logos from https://www.cncf.io/brand-guidelines/#logo?

Reply:
I have used the logo mentioned in the "Variations" section in the brand guidelines for the dark mode, light mode logo is the same. 

Should I change the dark mode logo to the primary one with a white background?

Reply:
Oh, what I mean is using the web image directly instead of downloading it into the repository.

Reply:
Oh okay! Makes sense, I'll do that.

Reply:
@BusyJay using the theme toggle needs the images to be hosted on GitHub as mentioned [here](https://stackoverflow.com/a/70200610/13308517)

Adding an external URL renders both the images in both the themes, like this:
![image](https://user-images.githubusercontent.com/54478821/148347336-4a03b85c-e51f-4c5e-a37c-74daf35eabe1.png)

Reply:
Reverted it back to the original(logos downloaded in the repository)

Reply:
Got it.